### PR TITLE
Split content and provide example for externaldb

### PR DESCRIPTION
**Pull request by mudler**
Splits the "Deployment" section which is becoming rather large in a new one "Advanced topics".
It also adds an example for providing the root password for externaldb connection (which just worked for me).

### CODE REVIEW

File: content/en/docs/Deployment/kubernetes-deploy.md [hunk 1]
```diff
@@ -128,7 +128,9 @@ cf-operator-quarks-secret-844844556b-xl9jq   1/1     Running   0          31s
 
 until all the pods are up & running.
 
-**Note:**
+
+{{% alert title="Note" %}}
+
 > The above `helm install` will generate many controllers spread over multiple pods inside the `cfo`
```
Comment (mook-as) [inline content/en/docs/Deployment/kubernetes-deploy.md:134]:
```suggestion
> The above `helm install` will generate many controllers spread over multiple pods inside the `cf-operator`
```

I realize you didn't change this, but still… it was wrong :)

File: content/en/docs/Deployment/kubernetes-deploy.md [hunk 2]
```diff
@@ -142,14 +144,20 @@ controllers to watch for CRD´s instances into the `kubecf` namespace.
 > If the `kubecf` namespace is deleted, but the operators are still running, they will no longer
 > know which namespace to watch. This can lead to problems, so make sure you also delete the pods
 > inside the `cfo` namespace, after deleting the `kubecf` namespace.
+{{% /alert %}}
+
+{{% alert title="Also" %}}
```
Comment (mook-as) [inline content/en/docs/Deployment/kubernetes-deploy.md:149]:
```suggestion
{{% alert title="Note" %}}
```
Having a section titled `Also` is kind of odd; just repeating it instead should be fine (given that it's just a callout, not a section header in the ToC).

File: content/en/docs/Deployment/kubernetes-deploy.md [hunk 3]
```diff
@@ -142,14 +144,20 @@ controllers to watch for CRD´s instances into the `kubecf` namespace.
...
 
-**Note:** 
-> how the namespace the operator is installed into (`cfo`) differs from the namespace the operator
+> How the namespace the operator is installed into (`cfo`) differs from the namespace the operator
```
Comment (mook-as) [inline content/en/docs/Deployment/kubernetes-deploy.md:151]:
```suggestion
> How the namespace the operator is installed into (`cf-operator`) differs from the namespace the operator
```

File: content/en/docs/Deployment/kubernetes-deploy.md [hunk 4]
```diff
@@ -276,131 +284,4 @@ cf auth admin "${admin_pass}"
...
-      credhub:
-        name: credhub
-        username: credhub-database-user
-        password: 5tJcIWHCR1QTLdfQDiN1Mz8K8jB+clD
-      autoscaler:
-        name: autoscaler
-        username: autoscaler-database-user
-        password: cYo2Tdi61N39mUjoEksXHuu3F7Dk58F
-```
+See [Advanced Topics](/docs/deployment/advanced-topics) for more configuration settings available in KubeCF
```
Comment (mook-as) [inline content/en/docs/Deployment/kubernetes-deploy.md:287]:
```suggestion
See [Advanced Topics](/docs/deployment/advanced-topics) for more configuration settings available in KubeCF.
```

**Review by mook-as — CHANGES_REQUESTED** [review]
I see that we have some duplicate content; a similar section is under [minikube docs](https://kubecf.io/docs/tutorials/deploy-minikube/#advanced-topics); perhaps we should make that point this way as well?

File: content/en/docs/Deployment/advanced-topics.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+weight: 20
+date: 2010-10-08
+description: >
+  Advanced topic and settings for KubeCF: Ingress, Diego and Eirini, ...
+---
+
+#### Diego vs Eirini
+
+Diego is the standard scheduler used by kubecf to deploy CF
+applications. Eirini is an alternative to Diego that follows a more
```
[the remark this answers is not in the document]
Comment (mudler) [reply content/en/docs/Deployment/advanced-topics.md]: :+1: Fixed in https://github.com/cloudfoundry-incubator/kubecf-docs/pull/35/commits/353d9bf2a5e01ae713f8398347c5df95eeba11b0

**Pull request merged**